ddtrace/llmobs/utils.py

```diff
@@ -21,6 +21,24 @@
 Message = TypedDict("Message", {"content": str, "role": str}, total=False)
 
 
+class AnnotationContext:
```

Comment:
Can we move this to `llmobs/_utils.py`? I'd rather users only add annotation contexts via `LLMObs.annotation_context()`